src/huntsman/pocs/observatory.py

```diff
@@ -297,9 +299,17 @@ def take_dark_fields(self,
 
         self.logger.debug(f'Using cameras {cameras_list}')
 
-        image_dir = self.get_config('directories.images')
+        filterwheel_events = dict()
 
-        self.logger.debug(f"Taking {n_darks} dark exposures with exposure times: {exptimes}")
+        self.logger.debug(f'Moving all camera filterwheels to blank filter')
+
+        # Move all the camera filterwheels to the blank position.
+        for camera in cameras_list.values():
+            if camera.filterwheel.current_filter != "blank":
+                filterwheel_event = camera.filterwheel.move_to("blank")
+                filterwheel_events[camera] = filterwheel_event
+        self.logger.debug('Waiting for all the filterwheels to be on blank filter...')
```

Comment:
Change message to `Waiting for all the filter wheels to move to the dark position`